### PR TITLE
Mkarnowski entra saml

### PR DESCRIPTION
**Pull request by mkarnowski**
Integration doc for Microsoft Entra ID and ngrok Dashboard SAML SSO.

### CODE REVIEW

File: docs/integrations/microsoft/sso-saml.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+description: Use SAML with Microsoft Entra ID to provide SSO to the ngrok Dashboard
```
Comment (russorat) [inline docs/integrations/microsoft/sso-saml.mdx:2]:
```suggestion
description: Use SAML with Microsoft Entra Verified ID to provide SSO to the ngrok Dashboard
```

File: docs/integrations/microsoft/sso-saml.mdx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+---
+
+# Microsoft Entra ID SSO (SAML)
```
Comment (russorat) [inline docs/integrations/microsoft/sso-saml.mdx:5]:
```suggestion
# Microsoft Entra Verified ID SSO (SAML)
```

File: docs/integrations/microsoft/sso-saml.mdx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+---
+
+:::tip TL;DR
+
+To secure access to ngrok Dashboard using Microsoft Entra ID Single Sign-On with SAML:
```
Comment (russorat) [inline docs/integrations/microsoft/sso-saml.mdx:11]:
```suggestion
To secure access to ngrok Dashboard using Microsoft Entra Verified ID Single Sign-On with SAML:
```

File: docs/integrations/microsoft/sso-saml.mdx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+1. [Configure Entra ID SSO](#configure-entra)
```
Comment (russorat) [inline docs/integrations/microsoft/sso-saml.mdx:13]:
```suggestion
1. [Configure Entra Verified ID SSO](#configure-entra)
```

File: docs/integrations/microsoft/sso-saml.mdx [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+1. [Configure ngrok](#configure-ngrok)
+1. [Test access to ngrok with Entra ID SSO](#test-sso)
+
+:::
+
+This article details how to configure Microsoft Entra ID as the primary Identity Provider for the ngrok Dashboard.
```
Comment (russorat) [inline docs/integrations/microsoft/sso-saml.mdx:19]:
```suggestion
This article details how to configure Microsoft Entra Verified ID as the primary Identity Provider for the ngrok Dashboard.
```

File: docs/integrations/microsoft/sso-saml.mdx [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+By integrating Entra ID SSO with ngrok, you can:
+
+- **Restrict access to the ngrok Dashboard** only to users authenticated via Entra ID
+- **Use Entra ID's Dashboard to facilitate access to the ngrok app**.
+
+## Requirements
+
+To configure the ngrok Dashboard with Microsoft Entra ID, you must have:
```
Comment (russorat) [inline docs/integrations/microsoft/sso-saml.mdx:27]:
```suggestion
To configure the ngrok Dashboard with Microsoft Entra Verified ID, you must have:
```

File: docs/integrations/microsoft/sso-saml.mdx (hunk 1 above)
Comment (mkarnowski) [reply docs/integrations/microsoft/sso-saml.mdx:2]: The Product itself is Microsoft Entra ID.  Verified ID is included in Entra ID.  In Azure the service is referred to as Entra ID.

I also believe that Verified ID is used for JWT.

Verified ID is currently included with any Microsoft Entra ID subscription, including Microsoft Entra ID Free, at no additional cost.  https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-verified-id#tabx67298482964d423da017e0eae7ba14b3

File: docs/integrations/microsoft/sso-saml.mdx (hunk 2 above)
Comment (mkarnowski) [reply docs/integrations/microsoft/sso-saml.mdx:5]: The Product itself is Microsoft Entra ID. Verified ID is included in Entra ID. In Azure the service is referred to as Entra ID.

I also believe that Verified ID is used for JWT.

Verified ID is currently included with any Microsoft Entra ID subscription, including Microsoft Entra ID Free, at no additional cost. https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-verified-id#tabx67298482964d423da017e0eae7ba14b3

File: docs/integrations/microsoft/sso-saml.mdx (hunk 4 above)
Comment (mkarnowski) [reply docs/integrations/microsoft/sso-saml.mdx:13]: The Product itself is Microsoft Entra ID. Verified ID is included in Entra ID. In Azure the service is referred to as Entra ID.

I also believe that Verified ID is used for JWT.

Verified ID is currently included with any Microsoft Entra ID subscription, including Microsoft Entra ID Free, at no additional cost. https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-verified-id#tabx67298482964d423da017e0eae7ba14b3

File: docs/integrations/microsoft/sso-saml.mdx (hunk 6 above)
Comment (mkarnowski) [reply docs/integrations/microsoft/sso-saml.mdx:27]: The Product itself is Microsoft Entra ID. Verified ID is included in Entra ID. In Azure the service is referred to as Entra ID.

I also believe that Verified ID is used for JWT.

Verified ID is currently included with any Microsoft Entra ID subscription, including Microsoft Entra ID Free, at no additional cost. https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-verified-id#tabx67298482964d423da017e0eae7ba14b3

**Review by russorat — APPROVED** [review]
LGTM!

**Pull request merged**